src/app/script.js

```diff
@@ -0,0 +1,48 @@
+import {fourOnFourMatrix as fourOnFourMatrix} from './4x4'
+import {thirtyTwoOnThirtyTwoMatrix as thirtyTwoOnThirtyTwoMatrix} from './32x32'
+
+let fourBtn = document.querySelector('.four'),
+    thirtyTwoBtn = document.querySelector('.thirty'),
+    imageBtn = document.querySelector('.image');
+
+fourBtn.addEventListener('click',function(){drawMatrix(fourOnFourMatrix, 4)});
+thirtyTwoBtn.addEventListener('click', function(){drawMatrix(thirtyTwoOnThirtyTwoMatrix, 32)});
+imageBtn.addEventListener('click', drawImage);
+
+function drawMatrix(colorMatrix, numOfCells){
+    let canvas = document.querySelector('.canvas'),
+    context  = canvas.getContext('2d'),
+    width = canvas.width,
+    height = canvas.height;
+
+    for(let i = 0; i <= width - width/numOfCells; i += width/numOfCells)
```

Comment:
lost curly braces in a loop. Please remove` width/numOfCells `calculation on top of this loop, use variables if width or numOfCells do not changes